x-pack/plugins/apm/public/components/app/RumDashboard/ClientMetrics/index.tsx

```diff
@@ -53,7 +53,7 @@ export function ClientMetrics() {
     (callApmApi) => {
       if (uxQuery) {
         return callApmApi({
-          pathname: '/api/apm/rum/client-metrics',
+          endpoint: 'GET /api/apm/rum/client-metrics',
```

Comment:
i am yet to take a look at that apm client contract you have created, will these types continue to work once we move ux code out of apm?

Reply:
You would have to create your own pseudo-router (`create_ux_api`) and client (`callUxApi`), but that was already the case, nothing in this PR changes what you need to do I think.